tpm2/tpm2.go

```diff
@@ -1838,3 +1838,44 @@ func RSADecrypt(rw io.ReadWriter, key tpmutil.Handle, password string, message [
 	}
 	return decodeRSADecrypt(resp)
 }
+
```

Comment:
We should also talk about authorization for this command, specifically that:
  - We need to use the "Lockout Authorization" value.
  - The value defaults to the empty password session auth.
  - That this value is set via a call to `HierarchyChangeAuth(HandleLockout)`